src/main/java/com/google/api/codegen/discovery/transformer/go/GoSampleMethodToViewTransformer.java

```diff
@@ -212,8 +223,17 @@ private void addStaticImports(SampleTransformerContext context, SymbolTable symb
     symbolTable.getNewSymbol("log");
     typeTable.saveNicknameFor("golang.org/x/net/context;;;");
     symbolTable.getNewSymbol("context");
-    typeTable.saveNicknameFor("golang.org/x/oauth2/google;;;");
-    symbolTable.getNewSymbol("google");
+    switch (config.authType()) {
+      case APPLICATION_DEFAULT_CREDENTIALS:
+        typeTable.saveNicknameFor("golang.org/x/oauth2/google;;;");
+        symbolTable.getNewSymbol("google");
+        break;
+      default:
+        typeTable.saveNicknameFor("errors;;;");
+        symbolTable.getNewSymbol("errors");
```

Comment:
Any reason these nickname/symbol pairs shouldn't be constructed by a single helper method?